index.d.ts

```diff
@@ -35,7 +35,7 @@ interface TypeOption {
 	//=> 'CDEHKM'
 	```
 	*/
-	type?: 'hex' | 'base64' | 'url-safe' | 'numeric' | 'distinguishable';
+	type?: 'hex' | 'base64' | 'url-safe' | 'numeric' | 'distinguishable' | 'ascii-printable';
```

Comment:
The doc comment needs an explanation of what ASCII printable means and some use-cases for it.

Reply:
Fixed!